src/helpers/shared/filters.js

```diff
@@ -0,0 +1,49 @@
+export const getFiltersFromUrl = (keys, defaultValues = {}) => {
+  const searchParams = new URLSearchParams(location.search);
+
+  let filters = {};
+
+  keys.forEach((key) => {
```

Comment:
You can use `map` to directly initialize the filters variable.

Reply:
@Hyperkid123 please, could you explain more? `array.map` returns an array, but how to make it set object entries? I can use `reduce` but I am not sure how to do this with `map` efficiently.